doxia-integration-tools/src/main/java/org/apache/maven/doxia/tools/DefaultSiteTool.java

```diff
@@ -560,39 +559,13 @@ private void populateModulesMenu(
         }
 
         // we require child modules and reactors to process module menu
-        if (project.getModules().size() > 0) {
+        List<MavenProject> projectModules = project.getCollectedProjects();
```

Comment:
This name is wrong. @gnodet does `#getCollectedProjects()` contain only direct children?

Reply:
Just tried, this is truly wrong:
![grafik](https://user-images.githubusercontent.com/573017/225149386-e07df943-3815-462b-b68e-72b8f4c743a1.png)

Reply:
This works `mavenSession.getProjectDependencyGraph().getDownstreamProjects(project, false)` when a624f1a5d113377f98a67eb698d8bbeffad4ce84 is applied.

Reply:
This does not work for aggregators without parent element. Reverting to `reactorProjects`.